cmd/maya-exporter/app/collector/pool/pool_test.go

```diff
@@ -1,64 +1,26 @@
 package pool
 
 import (
-	"errors"
 	"fmt"
-	"io"
-	"io/ioutil"
-	"net/http"
-	"net/http/httptest"
 	"regexp"
-	"strconv"
-	"strings"
-	"sync"
 	"testing"
-	"time"
 
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/openebs/maya/cmd/maya-exporter/app/collector/mock"
+	"github.com/openebs/maya/pkg/util"
```

Comment:
Avoid using util package
It will be removed in near future.

Reply:
@AmitKumarDas so should i rewrite the functions that i am reusing again into another package ?

Reply:
@utkarshmani1997 have provided comments later in this code. Have a look.